content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -2,24 +2,35 @@
 
 _New in DVC 2.0_
 
-To track successive steps in a longer experiment, you can register checkpoints
-from your code at runtime. This is especially helpful in machine learning, for
-example to track the progress in deep learning techniques such as evolving
-neural networks.
-
-_Checkpoint experiments_ track a series of variations (the checkpoints) and
-their execution can be stopped and resumed as needed. You interact with them
-using the `--rev` and `--reset` options of `dvc exp run` (see also the
-`checkpoint` field in `dvc.yaml` `outs`). They can help you
+To track successive steps in a longer machine learning experiment, you can
+register checkpoints from your code at runtime, for example to track the
+progress with deep learning techniques. They can help you
 
 - implement the best practice in deep learning to save your model weights as
   checkpoints.
 - track all code and data changes corresponding to the checkpoints.
 - see when metrics start diverging and revert to the optimal checkpoint.
 - automate the process of tracking every training epoch.
 
-> Experiments and checkpoints are [implemented](/blog/experiment-refs) with
-> hidden Git experiment commits branches.
+Checkpoint [execution] can be stopped and resumed as needed. You interact with
+them using the `--rev` and `--reset` options of `dvc exp run` (see also the
+`checkpoint` field in `dvc.yaml` `outs`).
```

Comment:
It might be better to use `dvc exp run --rev` to link directly to the option. Might need rephrasing though.

Reply:
I tend to agree but since there are 2 options here it would be too much to link both that way or weird to only link one. Could link them manually I guess... I'm not sure it's needed.

Please approve if there's no major feedback 🙂 Thanks @iesahin